addons/mail/static/tests/discuss/core/web/discuss_tests.js

```diff
@@ -199,4 +199,5 @@ QUnit.test("no conversation selected when opening non-existing channel in discus
     const { openDiscuss } = await start();
     await openDiscuss(200); // non-existing id
     await contains("h4", { text: "No conversation selected." });
+    await click(".o-mail-DiscussSidebar .btn", { text: "Channels" }); // check no crash
```

Comment:
It would be nice to test what happens after the click, I'm not sure if this is guaranteed to give enough time for the crash to happen as it is